### PR TITLE
Add measurement completion

### PR DESCRIPTION
**Pull request by bthesorceror**
- when used in logical == operation
- when a paramater for a function

### CODE REVIEW

File: src/handlers/completion.rs [hunk 1]
```diff
@@ -284,32 +421,46 @@ async fn find_completions(
     })
 }
 
-fn new_string_arg_completion(value: String) -> CompletionItem {
+fn new_string_arg_completion(
+    value: String,
+    trigger: Option<String>,
+) -> CompletionItem {
+    let trigger = trigger.unwrap_or_else(|| "".to_string());
```
Comment (jsteenb2) [inline src/handlers/completion.rs:428]: could you help me me understand what is happening here?

File: src/protocol/properties.rs [hunk 2]
```diff
@@ -235,6 +235,7 @@ impl Default for CompletionOptions {
                 ":".to_string(),
                 "(".to_string(),
                 ",".to_string(),
+                "\"".to_string(),
```
Comment (jsteenb2) [inline src/protocol/properties.rs:238]: I'm curious when this trigger occurs?

File: src/handlers/completion.rs (hunk 1 above)
Comment (bthesorceror) [reply src/handlers/completion.rs:428]: basically just says if trigger coming from the params is `None` then default it to an empty string

File: src/protocol/properties.rs (hunk 2 above)
Comment (bthesorceror) [reply src/protocol/properties.rs:238]: i.e. when I type `r._measurement == "` then it triggers a completion call

Comment (jsteenb2) [reply src/protocol/properties.rs:238]: ahh ok, so the triggered character could be `\"` in that case?

**Pull request merged**